### PR TITLE
Update config.sample

### PR DESCRIPTION
**Pull request by SaliniVenate**
Update the config.sample to include new sections and also move it to camera-streaming-daemon/samples/files directory.

### CODE REVIEW

File: samples/files/config.sample [hunk 1]
```diff
@@ -58,3 +58,21 @@
 #       and /dev/video456 can't be exported, set:
 #       blacklist = video123,video456
 #       Default: <empty>
+#
+# Section [uri]:
+#
+# Keys:
+#   videox
```
Comment (lbegani) [inline samples/files/config.sample]: videox can be replaced with <`camera-device-id`>

File: samples/files/config.sample [hunk 2]
```diff
@@ -58,3 +58,21 @@
...
+#       Absolute path to the location where the camera defenition file for videox is hosted
```
Comment (lbegani) [inline samples/files/config.sample]: >  Absolute path to the location where the camera defenition file for videox is hosted

URI for the Camera Definition File of the camera device <`camera-device-id`>

File: samples/files/config.sample [hunk 3]
```diff
@@ -58,3 +58,21 @@
...
+#       
+# Section [imgcap]
+#    
+# Keys:
+#   location
+#       The local path where captured image will be stored.
```
Comment (lbegani) [inline samples/files/config.sample]: > The local path where captured image will be stored.

The local path with write permission where captured images will be saved. For e.g :  /tmp/

**Comment by SaliniVenate** [comment]
I've made the changes.

File: samples/files/config.sample [hunk 4]
```diff
@@ -53,8 +53,26 @@
 #
 # Keys:
 #   Blacklist
-#       Coma separated list of /dev/video devices that won't be exported by
-#       camera-streaming-daemon. If the video streams from cameras /dev/video123
-#       and /dev/video456 can't be exported, set:
-#       blacklist = video123,video456
-#       Default: <empty>
```
Comment (lbegani) [inline samples/files/config.sample]: Any reason for change here?

Comment (SaliniVenate) [reply samples/files/config.sample]: Its for alignment.

**Pull request merged**